docs/developer-docs/latest/developer-resources/content-api/integrations/go.md

```diff
@@ -0,0 +1,324 @@
+        "name": "French Food",
+        "created_by": 1,
+        "updated_by": 1,
+        "created_at": "2021-03-31T11:36:23.164Z",
+        "updated_at": "2021-03-31T11:36:23.172Z"
+      },
+
+      {
+        "id": 2,
+        "name": "Brunch ",
+        "published_at": "2021-03-07T10:10:19.608Z",
+        "created_at": "2021-03-07T10:10:14.477Z",
+        "updated_at": "2021-03-07T10:10:19.649Z"
+}
+    ]
+  }
+]
+```
+
+### Example
+
+
+```go
+ package main
+import(
+    "fmt"
+    "io/ioutil"
+    "net/http"
+)
+
+func main() {
+    getD()
+}
+
+func getD() {
+    fmt.Println("Getting data...")
+    res, error := http.Get("http://localhost:1337/restaurants")
+    if error != nil {
+        fmt.Printf("The HTTP request failed with error %s\n", error)
+    } else {
+        data, _ := ioutil.ReadAll(res.Body)
+        fmt.Println(string(data))
+    }
+
+}
+```
+### POST Request your collection type
+
+Execute a `POST` request on the `restaurant` Collection Type in order to create a restaurant.
+
+Be sure that you activated the `create` permission for the `restaurant` Collection Type and the `find` permission for the `category` Collection type.
+
+
+_Request_
+
+```go
+  postRest, _ := json.Marshal(map[string]string{
+		  "name":  "Nwanyi Igbo",
+		  "description": "This is a very nice place to eat naive soup",
+		  })
+	   responseBody := bytes.NewBuffer(postRest)
+	   resp, error := http.Post("http://localhost:1337/restaurants", "application/json", responseBody)
+```
+
+_Response_
+
+```json
+{
+  "id": 2,
+  "name": "Nwanyi Igbo",
+  "description": "This is a very nice place to eat naive soup",
+  "created_at": "2021-03-04T09:57:11.669Z",
+  "updated_at": "2021-04-04T09:57:11.669Z",
+  "categories": [  ]
+}
+```
+
+### Example
+
+```go
+package main
+import(
+     "bytes"
+    "encoding/json"
+    "fmt"
+    "io/ioutil"
+    "log"
+    "net/http"
+)
+
+func main() {
+    postD()
+}
+
+func getD() {
+    fmt.Println("Getting data...")
+    resp, error := http.Get("http://localhost:1337/restaurants")
+    if error != nil {
+        fmt.Printf("The HTTP request failed with error %s\n", error)
+    } else {
+        data, _ := ioutil.ReadAll(resp.Body)
+        fmt.Println(string(data))
+    }
+
+}
+
+func postD() {
+
+	fmt.Println("Posting  data...")
+	//Encode the data
+	   postRest, _ := json.Marshal(map[string]string{
+		  "name":  "Nwanyi Igbo",
+		  "description": "This is a very nice place to eat naive soup",
+		  
+		  
+	   })
+	   
+	   responseBody := bytes.NewBuffer(postRest)
+	//Leverage Go's HTTP Post function to make request
+	   resp, error := http.Post("http://localhost:1337/restaurants", "application/json", responseBody)
+	//Handle Error
+	   if error != nil {
+		  log.Fatalf("An Error Occured %v", error)
+	   }
+	   defer resp.Body.Close()
+	//Read the response body
+	   body, error := ioutil.ReadAll(resp.Body)
+	   if error != nil {
+		  log.Fatalln(error)
+	   }
+	   
+	   fmt.Println(string(body))
+	 
+    }
+
+```	  
```

Comment:
Same here, can you indent the code? And add a space in the comments like this `// Handle Error`